lib/archethic/contracts/interpreter/action_interpreter.ex

```diff
@@ -13,13 +13,15 @@ defmodule Archethic.Contracts.Interpreter.ActionInterpreter do
   @doc """
   Parse the given node and return the trigger and the actions block.
   """
-  @spec parse(any()) :: {:ok, atom(), any()} | {:error, any(), String.t()}
-  def parse({{:atom, "actions"}, _, [keyword, [do: block]]}) do
+  @spec parse(any(), list) :: {:ok, atom(), any()} | {:error, any(), String.t()}
+  def parse(_, _ \\ [])
```

Comment:
I think this default value is not necessary since the function_key are always passed as parameter